backend/core/src/auth/dto/user-filter-type-to-field-map.ts

```diff
@@ -1,6 +1,6 @@
 import { UserFilterKeys } from "../types/user-filter-keys"
 
 export const userFilterTypeToFieldMap: Record<keyof typeof UserFilterKeys, string> = {
-  isPartner: "user_roles.isPartner",
-  isPortalUser: "user_roles",
+  isPartner: "userRoles.isPartner",
+  isPortalUser: "userRoles",
```

Comment:
I'm not 100% sure why this change was needed. It was required to get my changes to work but is not in the original change in bloom core. If anyone knows why this could be or if I missed some change it would be much appreciated

Reply:
It makes sense that it was `user_roles`, because the old `_getQb` function specified that as the alias. Looking at core/dev now and it does match what you have here, and it matches the join as set in the `getQb` function in `backend/core/src/auth/repositories/user-repository.ts`. So I think we're good.